UDPRenderer.pde

```diff
@@ -0,0 +1,59 @@
+import hypermedia.net.*;
+
+class UDPRenderer {
+  
+  private UDP udp;
+  private Config config;
+  
+  private Grid grid;
+  
+  UDPRenderer(Config config) {
+    this.config = config;
+    
+    udp = new UDP(this, config.localPort);
+    //udp.log(true);     //printout the connection activity
+    udp.listen(true);
+  }
+  
+  void renderGameState(TetrisGame currentGame) {
+    if (currentGame.isGameOver()) {
+      return;
```

Comment:
Could send an explicitly empty grid to make sure the display content is cleared between games.